docs/src/files.md

```diff
@@ -7,33 +7,12 @@ end
 
 # FASTX formatted files
 
-### Validate files
-The functions `validate_fasta` and `validate_fastq` can be used to check if an `IO`
-contains data that can be read as FASTX.
-They return `nothing` if the IO is correctly formatted, and another value if not.
-
-They are significantly faster than parsing the whole file into records,
-and are memory efficient.
-Be aware that the validators mutate the IO by reading it, so make sure to reset the IO before using it to parse FASTX files.
-
-```jldoctest
-julia> io = IOBuffer(">header\r\nAGG\nKK");
-
-julia> validate_fasta(io) === nothing
-true
-
-julia> read(io) # NB: IO is now exhausted
-UInt8[]
-
-julia> validate_fastq(IOBuffer("@header\nTTT\n+\njkm")) === nothing
-true
-```
-
-### Readers and writers
+### Readers and writers - basics
 A `Reader` and a `Writer` are structs that wrap an IO, and allows efficient reading/writing of FASTX `Record`s.
-For FASTA, use `FASTA.Reader` and `FASTA.Writer`, and for FASTQ - well I'm sure you've guessed it.
+For FASTA, use `FASTAReader` and `FASTAWriter`, and for FASTQ - well I'm sure you've guessed it.
```

Comment:
Looks like these changes are duplicates from #113 . If you branched from there, just be on the lookout for merge conflicts.